app/uk/gov/hmrc/cataloguefrontend/connector/TeamsAndRepositoriesConnector.scala

```diff
@@ -58,6 +58,36 @@ object RepoType {
     }
 }
 
+sealed trait ServiceType {def toString: String}
+
+case object FrontendService extends   ServiceType { override val toString = "FrontendService" }
```

Comment:
The instance's should move into `ServiceType` companion object - for namespace. You probably just need to call them `Frontend` and `Backend` (the context is `ServiceType`)